### PR TITLE
[ci skip] Update CONTRIBUTING.rst

### PR DESCRIPTION
**Pull request by tnecnivkcots**
changes from the already closed pull request #1415

@niklassiemer I hope now there are any things we have missed.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -357,9 +357,9 @@ The tag format consists of a tag_prefix (<package name>-) and the release versio
   This Workflow first installs all dependencies, then allows for future versions of the dependencies and builds the package. After that the package is published to `pip`_.
 3. Automatically create conda-forge package
   This release than is recognized by a conda-forge bot, which triggers a new pull request for the conda-forge package and merges automatically if all tests pass.
-  These tests are defined as `GitHub-Action-Workflows`_ and are triggered for every new pull request. More information can be found in the next chapter.
+  Check if all tests as explained in the next chapter have passed
```
Comment (niklassiemer) [inline CONTRIBUTING.rst:360]: Move this to the very beginning of the section, should be done before the tagging.

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -496,7 +496,7 @@ Now we know that the job class is indeed available in the project and can be ins
 
 Submitting and running a job on the queue, is essentially the same as saving a job in one notebook, but loading and executing it in another notebook.
 
-In **a new notebook** , load the job that you just saved, using its job id. You may or may not import the module (here, ``pyiron_conntirb``):
+In **a new notebook** , load the job that you just saved, using its job id. You may or may not import the module (here, `pyiron_conntirb`):
```
Comment (niklassiemer) [inline CONTRIBUTING.rst:499]:
```suggestion
In **a new notebook** , load the job that you just saved, using its job id. You may or may not import the module (here, `pyiron_contrib`):
```

**Review by niklassiemer — COMMENTED** [review]
I after all  found a few things xD

File: CONTRIBUTING.rst (hunk 1 above)
Comment (tnecnivkcots) [reply CONTRIBUTING.rst:360]: The complete point 3 or just this sentence?

Comment (tnecnivkcots) [reply CONTRIBUTING.rst:360]: I expect you want to have the complete point 3 as point 1. Am I right?

Comment (tnecnivkcots) [reply CONTRIBUTING.rst]: I have now moved the single sentence to the top of the section.

**Pull request merged**